src/Client.ts

```diff
@@ -16,9 +16,20 @@ import ClientCommands from "./commands/ClientCommands";
  * Lineage 2 Client main class
  */
 export default class Client extends ClientCommands {
-  LoginClient = new LoginClient();
+  LoginClient: LoginClient | null = new LoginClient();
   GameClient = new GameClient();
 
+  constructor() {
+    super();
+
+    this.once("PacketReceived", "KeyPacket", () => {
+      setImmediate(() => {
```

Comment:
### setImmediate
> Non-standard: This feature is non-standard and is not on a standards track. Do not use it on production sites facing the Web: it will not work for every user. There may also be large incompatibilities between implementations and the behavior may change in the future.
(https://developer.mozilla.org/en-US/docs/Web/API/Window/setImmediate)

I would suggest replacing with setTimeout(fn, 0), or simply remove it if we don't need an async action here.

Reply:
@npetrovski I use it in some other places as well, there is a shim if we need to run inside browsers, I did it asynchronously so event handler is not removing itself.